engine/solution_test.go

```diff
@@ -68,17 +68,17 @@ func TestSolution_MarshalJSON(t *testing.T) {
 	}
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			testJson, _ := tt.test.Reader()
```

Comment:
I don't really care about linting in test code, can you reset the change of this file?

Reply:
Does it hurt? What's the purpose of this naming? I just feel better if it passed the linting